packages/application/src/application/use-cases/submit-governance-review/subscribe-governance-reviews.service.ts

```diff
@@ -116,7 +116,7 @@ async function fetchReviews(client: IGithubClient, logger: Logger, application:
 function findApprovedReview(reviews: any[], logger: Logger, applicationId: string): any | undefined {
   const approvedReview = reviews.find(
     (review) =>
-      review.state === 'APPROVED' && config.GOVERNANCE_TEAM_GITHUB_HANDLES.includes(review?.user?.login as string),
+      review.state === 'DC_ALLOCATED' && config.GOVERNANCE_TEAM_GITHUB_HANDLES.includes(review?.user?.login as string),
```

Comment:
No. This is a PR from github, nothing to do with our states.

```suggestion
      review.state === 'APPROVED' && config.GOVERNANCE_TEAM_GITHUB_HANDLES.includes(review?.user?.login as string),
```